src/commands/cont.rs

```diff
@@ -47,3 +66,61 @@ impl ContinueCommand {
         Ok(())
     }
 }
+
+fn get_time_to_continue(
```

Comment:
No abbreviations

```suggestion
fn get_time_entry_to_continue(
```

Reply:
Yeah this is a typo, I intended to call it `get_time_entry_to_continue`